cmd/kube_list.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/exoscale/cli/table"
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+// kubeListCmd represents the kube list command
+var kubeListCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List running Kubernetes cluster instances",
+	Aliases: gListAlias,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return listKubeInstances()
+	},
+}
+
+func listKubeInstances() error {
+	vms, err := cs.ListWithContext(gContext, &egoscale.VirtualMachine{})
```

Comment:
with egoscale 0.13, you'll have more options here.

```go
vms, err := cs.ListWithContext(gContext, &egoscale.ListVirtualMachines{
    Tag: []egoscale.ResourceTag{{
        Key: "managedby",
        Value: "exokube",
    }}
})
```

Reply:
in 0.12, you'll have to enrich `VirtualMachine.ListRequest()` response.

Reply:
OK, I've put a `FIXME` to update the code once we reach egoscale 0.13.